lbrynet/daemon/DaemonCLI.py

```diff
@@ -38,8 +38,17 @@ def set_flag_vals(flag_names, parsed_args):
 
 
 def main():
-    if len(sys.argv[1:]):
-        method, args = sys.argv[1], sys.argv[2:]
+    argv = sys.argv[1:]
+
+    # check if a config file ha sbeen specified. If so, shift
+    # all the arguments so that the parsing can continue without
+    # noticing
+    if sys.argv[1] == "--conf":
```

Comment:
make sure you document that `--conf` must be the very first thing on the command line

Reply:
I am documenting it as part of the print_help() usage example. Let me know if it's enough